### ISSUE TITLE
[install script][xenial] Copied Ubuntu terminal setting not in effect

### ISSUE THREAD

**Issue by 130s** [130s/hut_10sqft#174]
`gnome-terminal` files are [copied into ~/.gconf](https://github.com/130s/hut_10sqft/blob/6f1e31e18170a6a61aa8424e7f5ae395d916452c/script/install_ubuntu_common.sh#L268_L272) directory, but it doesn't seem to be taking effect on p50.

[This thread](https://askubuntu.com/questions/628122/how-can-i-set-the-background-color-of-gnome-terminal-using-gconftool-2) may be suggesting that the same config might be no longer in use on 16.04...

**Comment by 130s** [comment]
Or something like [this](https://askubuntu.com/questions/868812/export-ubuntu-16-04-terminal-profile).

**Comment by 130s** [comment]
Revisiting this on p16s. https://github.com/kinu-garage/hut_10sqft/blob/develop/src/hut_10sqft/init_setup.py#L590 
is likely wrong -- symlinking `gnome-system-monitor.desktop` as `~/.gconf/apps` doesn't look right. On `serval` where the configs are functioning (which doesn't mean the config is taken from `~/.gconf` dir though) I see:

```
~/.gconf/apps$ tree -L 1
.
├── %gconf.xml
├── gnome-terminal
├── gnome-terminal.config.tgz
├── gnome-terminal.config.tgz.1
├── gnome-terminal.default
└── nm-applet
```

Rebooting OS to verify.

**Comment by 130s** [comment]
TLDR; Verified the following successfully copies the configs onto a new host.

Solution using `dconf`, not `gconf` https://askubuntu.com/questions/631481/gnome-terminal-profiles-are-not-being-loaded/694670#694670

Visual of terminal
```
$ dconf dump /org/gnome/terminal/legacy/profiles:/ | grep -e "\[\:\|visible-name"
[:b1dcc9dd-5262-4d8d-a863-c897e6d979b9]
[:fff0beb5-c6c9-42da-b71c-3f9ff4df9f36]

$ export DCONF_DUMPED_FILE=terminal_visible-name_1_130s-serval_20230327.dconf
$ dconf dump /org/gnome/terminal/legacy/profiles:/:b1dcc9dd-5262-4d8d-a863-c897e6d979b9/ > $DCONF_DUMPED_FILE
$ dconf load /org/gnome/terminal/legacy/profiles:/:b1dcc9dd-5262-4d8d-a863-c897e6d979b9/ < $DCONF_DUMPED_FILE

$ export DCONF_DUMPED_FILE=terminal_visible-name_2_130s-serval_20230327.dconf
$ dconf dump /org/gnome/terminal/legacy/profiles:/:fff0beb5-c6c9-42da-b71c-3f9ff4df9f36/ > $DCONF_DUMPED_FILE
$ dconf load /org/gnome/terminal/legacy/profiles:/:fff0beb5-c6c9-42da-b71c-3f9ff4df9f36/ < $DCONF_DUMPED_FILE
```

Shortcut keys
```
export DCONF_DUMPED_FILE=terminal_shortcuts_130s-serval_20230327.dconf
$ dconf dump /org/gnome/terminal/legacy/keybindings/ > $DCONF_DUMPED_FILE
$ dconf load /org/gnome/terminal/legacy/keybindings/ < $DCONF_DUMPED_FILE
```

**Comment by 130s** [comment]
https://github.com/kinu-garage/hut_10sqft/issues/174#issuecomment-1486187615
> On `serval` where the configs are functioning (which doesn't mean the config is taken from `~/.gconf` dir though) I see:
> 
> ```
> ~/.gconf/apps$ tree -L 1
> .
> ├── %gconf.xml
> ├── gnome-terminal
> ├── gnome-terminal.config.tgz
> ├── gnome-terminal.config.tgz.1
> ├── gnome-terminal.default
> └── nm-applet
> ```
> 
> Rebooting OS to verify.

On 24.04 https://github.com/kinu-garage/hut_10sqft/issues/1208#issuecomment-2995044640, that doesn't seem true -- I just manually set up copy and paste behavior on Terminal app but I don't see any files in that dir:
```
$ ll -a ~/.gconf
total 8.0K
drwxrwxr-x  2 n130s n130s 4.0K Jun 23 16:10 ./
drwxr-xr-x 22 n130s n130s 4.0K Jun 23 15:40 ../
```

**Comment by 130s** [comment]
While manually changing the terminal's transparency (which is what I want to apply by the premade config files), I saw these on the terminal:

```
/org/gnome/terminal/legacy/profiles:/:52522510-a00d-4b8f-a285-8ee2937237ff/background-transparency-percent
  36
/org/gnome/terminal/legacy/profiles:/:52522510-a00d-4b8f-a285-8ee2937237ff/background-transparency-percent
  35
/org/gnome/terminal/legacy/profiles:/:52522510-a00d-4b8f-a285-8ee2937237ff/background-transparency-percent
  34
/org/gnome/terminal/legacy/profiles:/:52522510-a00d-4b8f-a285-8ee2937237ff/background-transparency-percent
  33
```
`52522510-a00d....` might be the ID of the profile I had to create on Terminal's setting, which is different from the ID I see in 

Then I changed the transparency value for "`unnamed`" profile, which I assume is used by default.
```
/org/gnome/terminal/legacy/profiles:/:b1dcc9dd-5262-4d8d-a863-c897e6d979b9/background-transparency-percent
  17
/org/gnome/terminal/legacy/profiles:/:b1dcc9dd-5262-4d8d-a863-c897e6d979b9/background-transparency-percent
  16
/org/gnome/terminal/legacy/profiles:/:b1dcc9dd-5262-4d8d-a863-c897e6d979b9/background-transparency-percent
  15
```
The ID matches with the one in https://github.com/kinu-garage/hut_10sqft/issues/174#issuecomment-1486206854!!! So I could've just load the pre-saved config using that ID `b1dcc9dd-5262-4d8d-a863-c897e6d979b9` assuming this is the one that `dconf` or terminal generates as a first profile?

Problem is that I deleted that unnamed profile before I noticed its ID match..

Experimenting to load to non-existent profile ID.
```
$ export DCONF_DUMPED_FILE=terminal_visible-name_1_130s-serval_20230327.dconf
$ dconf load /org/gnome/terminal/legacy/profiles:/:b1dcc9dd-5262-4d8d-a863-c897e6d979b9/ < $DCONF_DUMPED_FILE
$ 
```
This one didn't return errors. Dumping it:
```
$ dconf dump /org/gnome/terminal/legacy/profiles:/:b1dcc9dd-5262-4d8d-a863-c897e6d979b9/ > 202506_terminal.log
$ cat 202506_terminal.log
[/]
background-transparency-percent=14
default-size-columns=110
default-size-rows=28
font='Monospace 11'
scrollback-lines=81920
use-custom-command=false
use-system-font=false
use-theme-transparency=false
use-transparent-background=true
```
This content aligns with the one that was loaded.

```
$ export ID_PROFILE_INUSE=52522510-a00d-4b8f-a285-8ee2937237ff
$ export FILE_PROFILE_CONF=202506_terminal.conf
$ dconf dump /org/gnome/terminal/legacy/profiles:/:$ID_PROFILE_INUSE/ > $FILE_PROFILE_CONF
$ cat $FILE_PROFILE_CONF
```
```
[/]
background-color='rgb(23,20,33)'
background-transparency-percent=13
foreground-color='rgb(208,207,204)'
use-theme-colors=false
use-theme-transparency=false
use-transparent-background=true
visible-name='130s'
```
Indeed the current setting is dumped. Manually modifying the dumped file and re-load it.
```
$ dconf load /org/gnome/terminal/legacy/profiles:/:$ID_PROFILE_INUSE/ < $FILE_PROFILE_CONF
```
I did see the change I added manually was applied after this cmd.

I guess implementing something like [askubuntu.com#954592/24203](https://askubuntu.com/a/954592/24203), automating the config setting may be possible. But at this point I found the value of doing that is so trivial so pause here.
